css/styles.theme.modes.css

```
@@ -255,21 +244,22 @@ html[data-bg-display="off"][data-theme="light"] body::after {
+  box-shadow: none;
 }
 
 [data-theme="light"] .scheme-weapon-item.is-selected.base-choice {
-  border-color: rgba(230, 124, 0, var(--alpha-95));
-  background: rgba(255, 235, 208, var(--alpha-98));
-  box-shadow:
+  border-color: rgba(var(--color-accent-rgb), var(--alpha-50));
+  background: rgba(var(--color-accent-rgb), var(--alpha-10));
+  box-shadow: none;
+}
     0 0 0 2px rgba(230, 124, 0, var(--alpha-45)),
```

Stylelint (17.7.0)

[error] Unknown word 0 (CssSyntaxError)

(CssSyntaxError)